### PR TITLE
Basic copy-to-s3 testing

### PR DESCRIPTION
**Pull request by def-**
### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  -

### CODE REVIEW

File: misc/python/materialize/checks/all_checks/copy_to_s3.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                """
+                """,
+                """
+                """,
+            ]
+        ]
+
+    def validate(self) -> Testdrive:
+        return Testdrive(
+            dedent(
```
[the remark this answers is not in the document]
Comment (def-) [reply misc/python/materialize/checks/all_checks/copy_to_s3.py:55]: There is a validation using testdrive's new `$ s3-verify-data`

[the remark this answers is not in the document]
Comment (nrainer-materialize) [reply misc/python/materialize/checks/all_checks/copy_to_s3.py:55]: Oh, yes, I was maybe reviewing an earlier version or just a single commit where the string was still empty.

**Review by nrainer-materialize — APPROVED** [review]
LGTM

File: misc/python/materialize/checks/all_checks/copy_to_s3.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (def-) [reply misc/python/materialize/checks/all_checks/copy_to_s3.py:55]: I think I made a mistake when splitting the commits, so you are probably right.

**Pull request merged**